go/pkg/pwdb/errors.go

```diff
@@ -7,13 +7,8 @@ import (
 	"pathwar.land/pathwar/v2/go/pkg/errcode"
 )
 
-func IsRecordNotFoundError(err error) bool {
-	return gorm.IsRecordNotFoundError(err) ||
-		gorm.IsRecordNotFoundError(errors.Unwrap(err))
-}
```

Comment:
I don't think that it will have the exact same result since on this current version of gorm, the gorm helper uses its own wrapping method -> https://github.com/jinzhu/gorm/blob/v1.9.16/errors.go#L25-L34

On gorm v2 (#644), the error handling is following golang standard, so this PR will probably have sense after #644 is merged (if not already managed in #644)